src/transformers/feature_extraction_utils.py

```diff
@@ -138,7 +139,13 @@ def convert_to_tensors(self, tensor_type: Optional[Union[str, TensorType]] = Non
                 raise ImportError("Unable to convert output to PyTorch tensors format, PyTorch is not installed.")
             import torch
 
-            as_tensor = torch.tensor
+            def as_tensor(value):
+                if isinstance(value, Sequence) and len(value) > 0:
+                    first_elem = value[0]
+                    if isinstance(first_elem, np.ndarray):
+                        value = np.array(value)
```

Comment:
```suggestion
                if isinstance(value, (list, tuple)) and len(value) > 0 and isinstance(value[0], np.ndarray):
                    value = np.array(value)
```
I'm not super enthusiastic about using `Sequence` here as it's wider than lists/tuples. We check for lists/tuples usually.